apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/DatasetsResourceTest.java

```diff
@@ -3795,6 +3812,12 @@ void findWithImageTruncation(JsonNode original, JsonNode expected, boolean trunc
                 assertThat(actualPage.total()).isEqualTo(expectedDatasetItems.size());
 
                 assertPage(expectedDatasetItems, actualPage.content());
+
+                for (int i = 0; i < expectedExperimentItems.size(); i++) {
```

Comment:
Minor: for a more endurable test, assert the size prior to the loop, in order to notice if the loop doesn't iterate.